lib/autoupdate/agent/installer.go

```diff
@@ -127,15 +126,15 @@ func (li *LocalInstaller) Remove(ctx context.Context, rev Revision) error {
 // Install a Teleport version directory in InstallDir.
 // This function is idempotent.
 // See Installer interface for additional specs.
-func (li *LocalInstaller) Install(ctx context.Context, rev Revision, template string) (err error) {
+func (li *LocalInstaller) Install(ctx context.Context, rev Revision, template string, baseURL string) (err error) {
```

Comment:
Can we remove the `template` param and move to it to a `Template` field on `LocalInstaller`? It could be set to `autoupdate.DefaultCDNURITemplate` in `NewLocalUpdater`, and directly set in tests.

Reply:
Make sense, I kept this one only for test in such case